daemon/uploads.py

```diff
@@ -43,7 +41,7 @@ async def do_upload(request: web.Request, task: tasks.Task):
     return
 
   task.change_progress(20, 'uploading files')  
-  error = await upload_asset_data(session, task, files, metadata_response)
+  error = await upload_asset_data(request.app['SESSION_UPLOADS'], task, files, metadata_response)
```

Comment:
Previously we were using the session_api_requests, now using specialized session for upload which has timeout increased to 60minutes (60*60 seconds)

Reply:
El error #1122 continue. It is as if you were trying to duplicate a key or main field in a database.